packages/libcomps/libcomps.spec

```diff
@@ -102,19 +102,17 @@ Python3 bindings for libcomps library.
 %{?scl:scl enable %{scl} - << \EOF}
 set -ex
 %autosetup -p1 -n %{name}-%{version}
-# workaround for https://github.com/rpm-software-management/libcomps/pull/64
-sed -i 's/EXACT//' libcomps/src/python/src/CMakeLists.txt
 
 # it can't detect our special PYTHONPATH and uses the compiled-in from the SCL Python
-%if 0%{?scl:1}
-sed -i "/OUTPUT_VARIABLE PYTHON_INSTALL_DIR/ s#))#).replace('rh/rh-python38', 'theforeman/tfm-pulpcore'))#" libcomps/src/python/src/CMakeLists.txt
-%endif
+#%if 0%{?scl:1}
+#sed -i "/OUTPUT_VARIABLE PYTHON_INSTALL_DIR/ s#))#).replace('rh/rh-python39', 'theforeman/tfm-pulpcore'))#" libcomps/src/python/src/CMakeLists.txt
+#%endif
```

Comment:
Since we're not building against el7 anymore, we can probably just drop any scl-related items.

Reply:
or, maybe not yet.  I don't know if we'll need to backport libcomps version updates to older pulp releases....  ignore this for now.

Reply:
yeah, that is why I'm keeping that, maybe we need to backport later

Reply:
Please keep as much as possible:

https://github.com/theforeman/pulpcore-packaging/pull/438#pullrequestreview-945273742

Reply:
I tried to keep as in the review for #438, and it failed to build for both versions